docs/use-cases/federated-learning/README.md

```diff
@@ -1,6 +1,38 @@
-# Federated learning
+# Federated learning on Google Cloud
```

Comment:
Should this be on GKE?

Reply:
We are mirroring what's written in the reference architecture: https://cloud.google.com/architecture/cross-silo-cross-device-federated-learning-google-cloud